src/components/apply/Apply.js

```diff
@@ -1,11 +1,87 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { getFormValues, initialize, touch, blur } from 'redux-form';
+import _ from 'lodash';
+import { Tabs, Tab } from 'material-ui/Tabs';
 
 import ApplicantForm from 'src/components/apply/ApplicantForm';
+import { addJointApplicant, selectJointApplicant } from 'src/actions';
 
-const Apply = () => (
-  <div>
-    <ApplicantForm />
-  </div>
-);
+const tabStyle = {
+  fontSize: '11pt',
+  paddingRight: 5,
+  paddingLeft: 5,
+  fontWeight: 400,
+};
 
-export default Apply;
+/**
+ * Helper function that initializes the form with the new values provided and executes the validators/normalizers
+ * for each untouched field as soon as its loaded, causing error messages to be displayed immediately.
+ */
+const initializeApplyForm = (formData, touchActionDispatcher, blurActionDispatcher) => {
+  // Actually insert the new values into the form
+  initialize('apply', formData, false);
+
+  // for each field that has values, simulate a `BLUR` event to trigger validators and normalizers
+  _.each(formData, (fieldValue, fieldName) => {
+    touchActionDispatcher('apply', fieldName);
+    blurActionDispatcher('apply', fieldName, fieldValue);
+  });
+};
+
+const Apply = ({
+  curValues, jointApplicantData, selectedApplicantId, jointApplicantCount,
+  addJointApplicant, selectJointApplicant, initialize, touch, blur
+}) => {
+  const applicantTabs = _.map(_.range(jointApplicantCount), i => {
+    return <Tab style={tabStyle} label={`Joint Applicant ${i + 1}`} key={i} value={i} />;
+  });
+  // Add an additional tab at the end to add additional joint applicants
+  const allTabs = [
+    ...applicantTabs,
+    <Tab style={tabStyle} label={'+ Add Joint Applicant'} key={applicantTabs.length} value={applicantTabs.length} />
+  ];
+
+  const numberOfApplicantsAllowed = 4;
+  const handleTabClick = index => {
+    if(index === jointApplicantCount) {
+      if(jointApplicantCount < numberOfApplicantsAllowed){
+        addJointApplicant();
+      }
+    }
+
+    selectJointApplicant(index, curValues);
```

Comment:
This is the same issue as above.  `selectJointApplicant()` creates a `SELECT_APPLICANT` action, while `initialize()` creates a `@@redux-form/INITIALIZE` action which deals with `redux-form` internals.

Although they are essentially part of the same event, I really don't see a way of merging these actions together.  It makes sense to me to have them separate as well since they're dealing with different slices of the state.